src/aux-runtime/runtime/AuxLibrary.spec.ts

```diff
@@ -17069,6 +17086,23 @@ describe('AuxLibrary', () => {
                 'data:image/png;base64,AQIDBAU='
             );
         });
+        it('should convert the given value to bytes when given an ArrayBuffer', () => {
+            const arrayBuffer = new ArrayBuffer(5);
+            const view = new Uint8Array(arrayBuffer);
+            view[0] = 1;
+            view[1] = 2;
+            view[2] = 3;
+            view[3] = 4;
+            view[4] = 5;
+            expect(library.api.bytes.toBase64Url(arrayBuffer)).toBe(
+                'data:image/png;base64,AQIDBAU='
+            );
+        });
+        it('should throw an error when not given Uint8Array or ArrayBuffer', () => {
+            expect(() => {
+                library.api.bytes.toBase64Url('invalid input');
```

Comment:
@TroyceGowdy `'invalid input'` needs to not be a string since strings are valid inputs for `bytes.toBase64Url()`